### PR TITLE
Fixes #14320 - packaging for client bootstrap script

### PR DESCRIPTION
**Pull request by mccun934**

### CODE REVIEW

**Comment by mccun934** [comment]
wrong branch, will resolve

**Comment by ehelms** [comment]
Master is the correct branch, you will just need to add this to the comps and releasers.conf file in the appropriate places.

File: client-bootstrap/client-bootstrap.spec [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# or any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+#
```
Comment (ehelms) [inline client-bootstrap/client-bootstrap.spec:17]: Copyright isn't needed at the top

File: client-bootstrap/client-bootstrap.spec [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Name:           client-bootstrap
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        Client bootstrap utility for Foreman and Katello
+
+Group:          System Environment/Base
+License:        LGPLv2
+URL:            http://www.katello.org
+Source0:        https://codeload.github.com/Katello/%{name}/tar.gz/%{version}
```
Comment (ehelms) [inline client-bootstrap/client-bootstrap.spec:27]: If you use this, the source has to be commited as '1.0.0' as if you did a wget from this location

**Pull request closed without merging**